### PR TITLE
Update .travis.yml

### PR DESCRIPTION
**Pull request by Lulalaby**

### CODE REVIEW

**Comment by appinteractive** [comment]
@Lulalaby why that change?

**Comment by Lulalaby** [comment]
@appinteractive because something was missing :)

**Comment by appinteractive** [comment]
Describe „something“ 😅

**Comment by Lulalaby** [comment]
I am not sure myself, because i can't test until a tagged release of master will be created.

File: .travis.yml [hunk 1]
```diff
@@ -39,6 +39,11 @@ after_success:
   - wget https://raw.githubusercontent.com/DiscordHooks/travis-ci-discord-webhook/master/send.sh
   - chmod +x send.sh
   - ./send.sh success $WEBHOOK_URL
+  - if [ $TRAVIS_BRANCH == "master" ] && [ $TRAVIS_EVENT_TYPE == "push" ]; then
```
Comment (roschaefer) [inline .travis.yml]: @Lulalaby if the intention is to notify our quality review for a new deployment, please put this in a `deploy` script (see the end of the file).

Comment (roschaefer) [reply .travis.yml]: Also you could send the URL instead of the word `staging`. So people just have to click on the link.

**Comment by Lulalaby** [comment]
Not really possible.
I have to create something new.

Am Di., 19. Feb. 2019, 12:39 hat Robert Schäfer <notifications@github.com>
geschrieben:

**Comment by Lulalaby** [comment]
Close #183

**Pull request merged**